content/en/blog/2020/new-deployment-model/index.md

```diff
@@ -0,0 +1,65 @@
+
+## External Control Plane Deployment model
+
+After installing Istio using the [default installation profile](/docs/setup/install/istioctl/#install-istio-using-the-default-profile), you will have an Istiod control plane installed in a single cluster like the diagram below:
+
+{{< image width="100%"
+    link="single-cluster.svg"
+    alt="Istio mesh in a single cluster"
+    title="Single cluster Istio mesh"
+    caption="Istio mesh in a single cluster"
+    >}}
+
+With this new deployment model in Istio 1.7, it is possible to run Istiod on a separate cluster (the `external control plane cluster`) from services, as shown in the diagram below. The `external control plane cluster` is owned by mesh operator while the mesh admin owns the cluster that has services deployed in the mesh. Mesh admin has no access to the `external control plane cluster`. For mesh operators, you can follow the [external istiod single cluster step by step guide](https://github.com/istio/istio/wiki/External-Istiod-single-cluster-steps) to explore more on this. This model was called central Istiod among Istio maintainers but external Istiod is the official name now.
+
+{{< image width="100%"
+    link="single-cluster-external-Istiod.svg"
+    alt="Istio mesh in a single cluster with Istiod outside"
+    title="Single cluster Istio mesh with Istiod outside"
+    caption="Single cluster Istio mesh with Istiod in an external control plane cluster"
+    >}}
+
+Mesh admins can expand the service mesh to multiple clusters, which are managed by the same Istiod running in the `external control plane cluster`. The `config cluster` also serves as the Istio configuration cluster, in addition to run services. The `external control plane cluster` reads Istio configurations from the `primary data plane cluster` and pushes them to both the `config cluster` and `remote cluster` per the diagram below.
+
+{{< image width="100%"
+    link="multiple-clusters-external-Istiod.svg"
+    title="Multicluster Istio mesh with Istiod outside"
+    caption="Multicluster Istio mesh with Istiod in an external control plane cluster"
+    >}}
+
+Mesh operators may further expand this deployment model to manage multiple Istio control planes from an external control plane cluster that runs multiple Istiod control planes:
+
+{{< image width="100%"
+    link="multiple-external-Istiods.svg"
+    alt="Istio meshes in single clusters with Istiod outside"
+    title="Multiple single clusters Istio meshes with Istiod outside"
+    caption="Multiple single clusters with multiple Istiod control planes in an external control plane cluster"
+    >}}
+
+The `external control plane cluster` can be used to host multiple Istiod control planes and each Istiod manages its own `config cluster`. Mesh operators could install another Istio mesh in the `external control plane cluster` and configure its `istio-ingress` gateway to route traffic from a config cluster to its corresponding Istiod control plane.
```

Comment:
I think this was actually talking about remote clusters, not config clusters.

Each Istiod manages its own set of remote clusters. Mesh operators can route requests from a remote cluster to its corresponding Istio control plane, for example by using Istio Gateway to handle the routing.

You could also potentially drop the whole discussion of using Istio to route requests to control planes, it is really cool but confusing (cue inception meme).

Reply:
Thanks, I struggle with this when writing the blog.  for single cluster scenario, the config cluster is also the remote cluster.  Yes it is indeed talking about remote clusters here.

I'll update the diagram that for single cluster, it is config + remote cluster.

I added a link to instructions about setting up Istio to route requests to istiod in case mesh operators want to learn more.  Hope this helps to clarify this a bit.